frontend/src/metabase/visualizations/lib/apply_axis.js

```diff
@@ -145,7 +145,9 @@ export function applyChartTimeseriesXAxis(
       const timestampFixed = moment(timestamp)
         .utcOffset(dataOffset)
         .format();
-      const { column, columnSettings } = chart.settings.column(dimensionColumn);
+      const { column, ...columnSettings } = chart.settings.column(
```

Comment:
Oh interesting, I though spreading `undefined` (which `columnSettings` would have been) would have crashed, but apparently you can spread `null`/`undefined`. I guess I can stop doing things like `{ ...(something || {}) }`

Reply:
Yeah! I just had the [same realization](https://github.com/metabase/metabase/pull/10996#discussion_r335158902) a few days ago!